libs/pChart2.1.3/class/pImage.class.php

```diff
@@ -166,6 +166,17 @@ function stroke($BrowserExpire=FALSE)
      header('Content-type: image/png');
      imagepng($this->Picture);
     }
+    
+   /* Updated code to send image as encoded data */
+   function strokeAsImageData()
```

Comment:
Would it be possible to the `ob_start` etc the StaticGraph class eg in a private method `strokeAsImageData`? Would be good if it was possible. Otherwise we always need to make sure to apply this patch when updating the pchart library etc.